doc-Deployment_Planning_Guide/topics/capacity.adoc

```diff
@@ -273,6 +276,12 @@ ifdef::miq[Managing Providers]
   After adding the provider, capacity and utilization data for your instances
   populate in a few minutes.
 
+[[data_collection_for_gce]]
+=== Data Collection for Google Compute Engine
+  
+{product-title} can collect capacity and utilization metrics from Google Compute Engine once Google Cloud Monitoring API is enabled. Visit the Google APIs API Manager site to enable the Google Cloud Monitoring API. Once enabled, it may take a few minutes to propgate to {product-title_short}.
```

Comment:
@cbudz Thanks for making the updates, looks good to me. I have a minor suggestion to avoid using the word API consecutively, thereby making it a bit easier to read. (It seems the Google APIs API Manager is called the Google API Console.) Also, worth including a link to the Console I think: 

"Visit the https://console.developers.google.com [Google API Console] to enable the Cloud Monitoring API. Once enabled, it may take a few minutes to **propagate** to CloudForms."

More importantly, it looks like the Google Cloud Monitoring API v2 is deprecated in favor of the new Stackdriver Monitoring API v3. (Although, the link under Cloud APIs in the API Console library still says Cloud Monitoring API, but the Google Cloud Platform product page and documentation redirects to Stackdriver) It'd be worth checking if CloudForms supports the Stackdriver Monitoring API v3 for C&U data collection (Guess it should for C&U to work -- as per Google, custom metrics sent to the v2beta2 API can no longer used for alerting and suggests migrating to Stackdriver asap). If so, I wonder if we need to update the above line to mention the Stackdriver Monitoring API v3 instead, or perhaps include a note about the change?